THANKS.md

```diff
@@ -13,3 +13,4 @@ Huge thanks to the following contributors (by github username). For the most up-
 - [tschieggm](https://github.com/tschieggm)
 - [zvirusz](https://github.com/zvirusz)
 - [isra17](https://github.com/isra17)
+- [juggernaut](https://github.com/juggernaut)
```

Comment:
This list should be in alphabetical order (the rest of it is, before the last two names)